.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,51 @@
+name: Publish package to PyPI
+
+on:
+  push:
+    tags:
+      - '*' # Push events to every tag not containing /
+
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.9"
+    - name: Install pypa/build
+      run: >-
+        python3 -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: python3 -m build
+    - name: Store the distribution packages
+      uses: actions/upload-artifact@v4
+      with:
+        name: python-package-distributions
+        path: dist/
+   publish-to-pypi:
+    name: >-
+      Publish to PyPI
+    if: startsWith(github.ref, 'refs/tags/')  # only publish to PyPI on tag pushes
+    needs:
+    - build
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/p/scalr  # Replace <package-name> with your PyPI project name
```

Comment:
Is this url belong(as i can see it belongs to other software) to us or will it create new one ?

Reply:
Thanks for noticing. Will replace with `pyscalr`